docs/guides/launch.md

```diff
@@ -54,290 +54,109 @@ access it at `http://127.0.0.1:3000/` under credentials `admin/admin`.
-```
-
-**Description**. This means that your bytecode config file has an empty entry: `"bytecode": "0x"`. This happens because
-your `zksync-2-dev/etc/system-contracts/package.json`'s dependency on `"@matterlabs/hardhat-zksync-solc"` is outdated.
-We don't expect this error to happen as we've updated to latest version which fixes the problem.
-
-**Solution**. Update your dependency and reinit:
-
-```
-yarn add -D @matterlabs/hardhat-zksync-solc # in the system-contracts folder
-zk clean --all && zk init
-```
-
-On the run, it moved from:
-
-```
-    "@matterlabs/hardhat-zksync-solc": "^0.3.14-beta.3",
-```
-
-to:
-
-```
-    "@matterlabs/hardhat-zksync-solc": "^0.3.15",
-```
-
-### Error: Bytecode length in 32-byte words must be odd
-
-**Problem**. `zk init` fails with an error similar to:
+Ensure that the necessary containers have been started and are functioning correctly to resolve the issue.
 
-```
-Successfully generated Typechain artifacts!
-Error: Error: Bytecode length in 32-byte words must be odd
-    at hashL2Bytecode (/Users/emilluta/code/zksync-2-dev/contracts/zksync/src/utils.ts:29:15)
-    at computeL2Create2Address (/Users/emilluta/code/zksync-2-dev/contracts/zksync/src/utils.ts:53:26)
-    at /Users/emilluta/code/zksync-2-dev/contracts/zksync/src/compileAndDeployLibs.ts:50:63
-    at step (/Users/emilluta/code/zksync-2-dev/contracts/zksync/src/compileAndDeployLibs.ts:33:23)
-    at Object.next (/Users/emilluta/code/zksync-2-dev/contracts/zksync/src/compileAndDeployLibs.ts:14:53)
-    at fulfilled (/Users/emilluta/code/zksync-2-dev/contracts/zksync/src/compileAndDeployLibs.ts:5:58)
-error Command failed with exit code 1.
-info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
-error Command failed.
-Exit code: 1
-Command: /Users/emilluta/.nvm/versions/node/v16.19.1/bin/node
-Arguments: /opt/homebrew/Cellar/yarn/1.22.19/libexec/lib/cli.js compile-and-deploy-libs
-Directory: /Users/emilluta/code/zksync-2-dev/contracts/zksync
-Output:
-
-info Visit https://yarnpkg.com/en/docs/cli/workspace for documentation about this command.
-error Command failed with exit code 1.
-info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
-Error: Child process exited with code 1
+```bash
+zkstack containers
 ```
 
-**Description**. This means that your bytecode config file has an empty entry: `"bytecode": "0x"`. This happens because
-your `zksync-2-dev/contracts/zksync/package.json`'s dependency on `"@matterlabs/hardhat-zksync-solc"` is outdated. We
-don't expect this error to happen as we've updated to latest version which fixes the problem.
+### Failed To Remove Volumes
 
-**Solution**. Update your dependency and reinit:
+#### Problem
 
-```
-yarn add -D @matterlabs/hardhat-zksync-solc # in the system-contracts folder
-zk clean --all && zk init
+```bash
+failed to remove path `/home/<user>/workspace/current/zksync-era/volumes`: Permission denied (os error 13)
 ```
```

Comment:
This whole section can be removed since #3140 got merged

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/3134/commits/3216bb10654f2143b2466aaa00ec89f52b02c8e5